bears/vcs/git/GitCommitBear.py

```diff
@@ -260,6 +268,25 @@ def check_body(self, body,
                                'Commit body lines should not exceed {} '
                                'characters.'.format(body_line_length))
 
```

Comment:
The code does not comply to PEP8.

*PEP8Bear, severity NORMAL, section `autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/bears/vcs/git/GitCommitBear.py
+++ b/bears/vcs/git/GitCommitBear.py
@@ -280,7 +280,7 @@
                     # look up dns
                     if email_dns_check and email_check:
                         email_check = not is_email(email, diagnose=True,
-                                               check_dns=True).ERROR_CODES
+                                                   check_dns=True).ERROR_CODES
                     if not email_check:
                         invalid_emails = True
                         result_message += ' ' + email + '\n'
```